third_party/docfx/templates/devsite/partials/namespaceSubtitle.tmpl.partial

```diff
@@ -1,37 +1,44 @@
-{{!Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.}}
-{{^isNamespace}}
-{{#inNamespace}}
-{{__global.namespacesInSubtitle}}
-{{/inNamespace}}
-{{/isNamespace}}
-{{#inModule}}
-{{__global.modulesInSubtitle}}
-{{/inModule}}
-{{#inClass}}
-{{__global.classesInSubtitle}}
-{{/inClass}}
-{{#inStruct}}
-{{__global.structsInSubtitle}}
-{{/inStruct}}
-{{#inInterface}}
-{{__global.interfacesInSubtitle}}
-{{/inInterface}}
-{{#inEnum}}
-{{__global.enumsInSubtitle}}
-{{/inEnum}}
-{{#inDelegate}}
-{{__global.delegatesInSubtitle}}
-{{/inDelegate}}
-{{#inFunction}}
-{{__global.functionsInSubtitle}}
-{{/inFunction}}
-{{#inVariable}}
-{{__global.variablesInSubtitle}}
-{{/inVariable}}
-{{#inTypeAlias}}
-{{__global.typeAliasesInSubtitle}}
-{{/inTypeAlias}}
-
-{{#inConst}}
-{{__global.constsInSubtitle}}
-{{/inConst}}
+{{!Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.}}
```

Comment:
Looks like there are whitespace changes in this file -- mind reverting those?

Reply:
Somewhere along I kept getting carriage returns which bugged me a lot and ended up templating this whole file with new spacing issue and I think some of it got caught in the goldens update.

Fixed it by running `git config --global core.whitespace cr-at-eol` which supposedly tells git to treat carriage returns as part of just whitespace. The updated file seems good now.